content/rs/security/tls-ssl.md

```diff
@@ -6,68 +6,82 @@ alwaysopen: false
-         ![database-tls-replica-certs](/images/rs/database-tls-replica-certs.png "Database TLS Configuration")
-1. Save the certificate ![icon_save](/images/rs/icon_save.png#no-click "Save")
-1. Repeat for any certificates or certificate authority (CA) you would like to use for authentication.
+1. Select **Add** ![Add](/images/rs/icon_add.png#no-click "Add")
+1. Paste your certificate or certificate authority (CA) into the text box.
+    ![database-tls-replica-certs](/images/rs/database-tls-replica-certs.png "Database TLS Configuration")
+1. Save the certificate. ![icon_save](/images/rs/icon_save.png#no-click "Save")
+1. Repeat for each client certificate you need to add.
+    - If your database uses Replica Of or Active-Active replication, you will need to add the syncer certificates for the participating clusters. The steps for each are below.
+1. Optional: To limit connections further to a subset of those with valid certificates, enforce **Subject Alternative Name** and enter authorized users separated with commas.
+1. Select **Update** at the bottom of the screen to save your configuration.
 
-{{< note >}}
-There are two considerations for replication authentication you should be aware of:
+## Enable TLS for Active-Active cluster connections
 
-1. The syncer certificates of the clusters that host the replica instances of the database must always be set when enabling a database for encryption.
-2. When using Active-Active, the syncer certificate for each cluster must be configured on the database.
+{{< note >}}
+You cannot enable or disable TLS after the Active-Active database is created, but you can change the TLS configuration.
 {{< /note >}}
 
-## Certificate Authentication for Active-Active Databases
+### Retrieve syncer certificates
 
-When you create a new CRDB, you can configure authentication for traffic between active-active databases using the same process for as replication traffic.
+1. For each participating cluster, copy the syncer certificate from the **general** settings tab.
 
-{{< note >}}
-You cannot enable or disable TLS after the CRDB is created, but you can change
-the TLS configuration.
-{{< /note >}}
+### Configure TLS certificates for Active-Active
 
-### Configuring TLS for CRDB communication
+1. During database creation (see [Create an Active-Active Geo-Replicated Database]({{<relref "content/rs/administering/creating-databases/create-active-active.md" >}}), select **Edit** from the **configuration** tab.
+1. Enable **TLS**.
+    - **Enforce client authentication** is selected by default. If you choose to change this option, you will enforce encryption without authentication.
+1. Select **Require TLS for CRDB communication only** from the dropdown menu.
+    ![crdb-tls-all](/images/rs/crdb-tls-all.png "crdb-tls-all")
+1. Select **Add** ![Add](/images/rs/icon_add.png#no-click "Add")
+1. Paste a syncer certificate into the text box.
```

Comment:
```suggestion
1. Paste certificate or CA to be used for the syncer into the text box.
```

Reply:
Is this more accurate?

Reply:
It is 👍

Reply:
I didn't realize the CA feature extended to syncer certificates? I thought it was only for client certificates? @AlonMagrafta can you verify?

Reply:
Using a CA for the syncer is possible, but complicated and needs more consideration in order to document. DOC-647 will track this work.